techniques/general/providing-single-point-control-slider.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+  <head>
+		<title>Providing a control slider that offers single point activation</title>
+		<link rel="stylesheet" type="text/css" href="../../css/editors.css"/>
+	</head>
+	<body>
+		<h1>Providing a control slider that offers single point activation</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p id="id"></p>
+			<p id="technology">general</p>
+			<p id="type">sufficient</p>
+		</section>
+		<section id="applicability">
+			<h2>When to use</h2>
+			<h3>Applicability</h3>
+			<p>Any technology that supports pointer input (e.g. supporting any or all of the following: mouse pointer, touch on touch screen or trackpad, stylus input, or laser pointer input).</p>
+		</section>
+		<section id="description">
+			<h2>Description</h2>		
+			<p>The objective of this technique is to ensure that users who have difficulties performing path-based gestures can operate a control slider. A control slider is a track with a &quot;thumb&quot; that you move along the track to set a value.</p>
+
+			<figure>
+				<img src="img/input-range.png" alt="A horizontal line with a round button halfway along the line, labelled volume." />
+				<figcaption>Screenshot of an example control slider showing the thumb in the middle of the track.</figcaption>
+			</figure>
+			
+			<p>It allows a user to set a value in a certain range, e.g. setting the volume, changing the hue value of a color, putting in the amount of money needed in a loan calculator, or picking a sum to be donated to a charity. A slider that requires path-base gestures would use swiping left or right to change the value, or dragging the thumb of the slider in a specific direction to the change the value.</p>
+
+        	<p>A simple fallback for activation without a path-based gesture is to make the control slider track clickable. This way, a value can be specified using single tap or click on the track.</p>
+     		<p>Providing controls (e.g., arrow buttons) as alternative also allows incrementing or decrementing the value with single pointer input. This allows for a more fine-grained setting of the value.</p>
+		
+      		<p class="note">On touch screen devices, author-supplied path-based gestures usually do not work when OS level assistive technologies (AT) like a built-in screenreader are turned on. This is because AT generally consumes path-based gestures so they would not reach the authored content. When custom controls are built on top of native controls, however, these may also be operable with AT gestures such as vertical swiping to change the value (see example 1).</p>
```

Comment:
"AT generally consumes" > "AT generally consume" ("AT" being plural?)

Reply:
I think of AT = Assistive Technology (singular)

Reply:
> I think of AT = Assistive Technology (singular)

but in the sentence just preceding it it's been set as plural `OS level assistive technologies (AT)`. should both be the same either way, i'd say.

Reply:
Updated to be consistent (singular).